R/benchmark.R

```diff
@@ -17,7 +17,7 @@
 #' a range of parameter combinations is handled by the runner, not the functions
 #' in the benchmark object.
 #'
-#' @section Parametrizing benchmarks:
+#' @section Parameterizing benchmarks:
```

Comment:
❤️

Reply:
Apparently "parametrizing" is allowable but more British? And I'd already written "parameterizing" elsewhere, so I changed this for consistency

Reply:
LOL I just assumed I mis-spelled it 🙈